src/webgpu/api/validation/encoding/cmds/dynamic_render_state.spec.ts

```diff
@@ -0,0 +1,299 @@
+export const description = `
+API validation tests for dynamic state commands (setViewport/ScissorRect/BlendColor...).
+
+TODO: test passing NaN values, or is that a WebIDL test?
+`;
+
+import { params } from '../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+
+import { ValidationTest } from './validation_test.js';
+
+interface ViewportCall {
+  x: number;
+  y: number;
+  w: number;
+  h: number;
+  minDepth: number;
+  maxDepth: number;
+}
+
+interface ScissorCall {
+  x:number,
+  y:number,
+  w:number,
+  h:number,
```

Comment:
Please `npx grunt fix`

Reply:
Whoops I did it locally but forgot to add it to the PR ^^